spec/lib/email_alert_criteria_spec.rb

```diff
@@ -0,0 +1,135 @@
+require "gds_api/test_helpers/content_item_helpers"
+
+def valid_content_item
+  content_item = content_item_for_base_path("/criteria-test")
+  content_item["base_path"] = "/criteria-test"
+  content_item["locale"] = "en"
+  content_item["details"].merge!("change_history" => [{
+    "note" => "updated!",
+    "public_updated_at" => content_item["public_updated_at"],
+  }])
+  content_item["publishing_app"] = "whitehall"
+  content_item["document_type"] = "news"
+  content_item["parent"] = %w[00f693d4-866a-4fe6-a8d6-09cd7db8980b]
+  content_item
+end
+
+def valid_content_item_no_parent
+  content_item = valid_content_item
+  content_item.delete("parent")
+  content_item
+end
+
+RSpec.describe EmailAlertCriteria do
+  include GdsApi::TestHelpers::ContentItemHelpers
+
+  describe "#would_trigger_alert?" do
+    context "when the content item has no title" do
+      let!(:content_item) { valid_content_item.except("title") }
+
+      it "should return false" do
+        expect(EmailAlertCriteria.new(content_item:).would_trigger_alert?).to be false
```

Comment:
And then `subject.would_trigger_alert?` in each test

Reply:
Done. Thanks!